src/Fixer/Internal/PhpUnitRequiresWithCaretFixer.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Internal;
+
+use PhpCsFixer\Fixer\AbstractPhpUnitFixer;
+use PhpCsFixer\Fixer\InternalFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @internal
+ */
+final class PhpUnitRequiresWithCaretFixer extends AbstractPhpUnitFixer implements InternalFixerInterface
+{
+    public function getName(): string
+    {
+        return 'PhpCsFixerInternal/'.parent::getName();
```

Comment:
I believe we rather need to update `php_unit_attributes` to add ^ when missing.
otherwise, most of ppl will fail on the migration even without being aware of that (the issue we faced ourselves with disappeared tests)